### PR TITLE
improve the documents

### PR DESCRIPTION
**Pull request by xiangao1**
## Fixes
Documents for the double loop.

## Changes proposed in this PR:
- main page: explain each step in the double loop; mention the Rankine cycle example.
- main page: include the design of the implementation
- bidder and tracker: add figure examples.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by xiangao1** [comment]
Hi @adowling2, I have revised the documentation as we discussed. Would you mind taking a look at them? Specifically, index.rst, Bidder.rst, Tracker.rst. Thanks!

File: idaes/apps/grid_integration/docs/index.rst [hunk 1]
```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
...
-(figure left), economic dispatch problems are solved every
-hour to clear the market and set generation targets for each resource. Each
-energy system then solves optimal control problems to track these dispatch signals from
-real-time markets constrained by (non)linear process models. Finally, the market
-calculates the settlements and pays the energy systems.
+wholesale electricity markets. This multiscale framework aims to:
+(1) elucidate complex relationships between resource dynamics and market dispatch
+(with uncertainty, beyond price-taker assumption);
+(2) predict the economic opportunities and market impacts of emerging
+technologies (e.g., H2 production, integrated energy systems);
```
Comment (adowling2) [inline idaes/apps/grid_integration/docs/index.rst]: H2 production --> hydrogen co-production

File: idaes/apps/grid_integration/docs/index.rst [hunk 2]
```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
...
+(3) guide conceptual design & retrofit to meet current and future power grid
```
Comment (adowling2) [inline idaes/apps/grid_integration/docs/index.rst]: & --> and

File: idaes/apps/grid_integration/docs/index.rst [hunk 3]
```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
...
+needs.
+
+The framework integrates optimal operations, and control of an energy system with
+market clearing via a high-fidelity Production Cost Model (PCM) Prescient which
+optimizes resource dispatch decisions across an entire transmission network.
+Specifically, in the day-ahead market loop, the following steps are modeled:
+(a) Forecast: energy systems perform market uncertainty forecasts, e.g., locational
+marginal prices (LMP).
+(b) Bid: energy systems utilize the uncertainty forecasts and solve optimization
+problems to derive optimal bidding strategies into the markets.
```
Comment (adowling2) [inline idaes/apps/grid_integration/docs/index.rst]: derive --> compute

File: idaes/apps/grid_integration/docs/index.rst [hunk 4]
```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
...
+calculates and pays the energy and ancillary service settlements to the participants.
+
+.. |doubleloop_implementation| image:: images/implementation.png
+  :width: 400
+  :alt: Alternative text
+  :align: middle
+
+|doubleloop_implementation|
+
+The design of framework's implementation is shown in the figure above. Specifically,
```
Comment (adowling2) [inline idaes/apps/grid_integration/docs/index.rst]: Replace the first two sentences with: "The above figure shows the implementation. The objects ``Forecaster``, ``Forecaster``, ``Bidder``, ``Tracker``, and ``Coordinator`` help manage information exchange within the multiscale integrated simulation framework.

File: idaes/apps/grid_integration/docs/index.rst [hunk 5]
```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
...
+we added the objects in red ``Forecaster``, ``Bidder``, ``Tracker``, and ``Coordinator``.
+The ``Bidder`` and ``Tracker`` take an IDAES model, and augment the model with
```
Comment (adowling2) [inline idaes/apps/grid_integration/docs/index.rst]: augments

**Review by adowling2 — COMMENTED** [review]
Looks great. A few small suggestions.

**Pull request merged**